app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
```

Comment:
- [ ] To make the associations work correctly in the belongs_to statement for user you should add the class as well, similar to what you did in the post model